web/src/types.ts

```diff
@@ -148,3 +148,9 @@ export type SidebarOptions = {
   // Filter options
   resourceNameFilter: string
 }
+
+// TODO (lizz): Centralize other redundant type definitions
```

Comment:
I think doing this now will make the diff too big to be useful. 🤓 I'll do this in a follow-up pr.